enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/replication/tx/ReplicatedTransactionStateMachine.java

```diff
@@ -45,6 +45,7 @@
 
 public class ReplicatedTransactionStateMachine<MEMBER> implements StateMachine
 {
+    static final byte HEADER_ID = 0;
```

Comment:
That is a valid point. The reason i didn't choose that route was that it would leak core-edge concepts in the kernel. And since we expect a really small namespace, i thought that clashes would be easy to avoid. Not really decided on that. On a side note, perhaps this value should also be "1", to avoid the case of accidentally having 0's everywhere, like you mention in the previous comment.